logging/cluster-logging-release-notes.adoc

```diff
@@ -10,47 +10,20 @@ toc::[]
 
 Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright's message].
 
-[id="openshift-logging-about-this-release-5-1"]
-
-== Red Hat OpenShift Logging 5.1
-
-.Supported versions
-
-OpenShift Logging 5.1 runs on {product-title} 4.7 and 4.8.
-
-.Advisories
-
-The following advisories are available for OpenShift Logging 5.1.x:
-
-* link:https://access.redhat.com/errata/RHBA-2021:3545[RHBA-2021:3545 - Bug Fix Advisory. Openshift Logging Bug Fix Release 5.1.2]
-* link:https://access.redhat.com/errata/RHBA-2021:2885[RHBA-2021:2885 - Bug Fix Advisory. Openshift Logging Bug Fix Release 5.1.1]
-* link:https://access.redhat.com/errata/RHBA-2021:2112[RHBA-2021:2112 - Bug Fix Advisory. OpenShift Logging Bug Fix Release 5.1.0]
+== Supported Versions
```

Comment:
@libander Needs an anchor ID for this section heading. Please refer to: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings